features_api/runtime/Dockerfile

```diff
@@ -4,6 +4,7 @@ WORKDIR /tmp
 
 COPY features_api/runtime /tmp/features
 RUN pip install "mangum>=0.14,<0.15" /tmp/features["psycopg-binary"] -t /asset --no-binary pydantic
+RUN pip install pygeofilter -t /asset
```

Comment:
why this? 🤔

Reply:
Lambda was complaining about this dependency once the proxy was enabled. Without the proxy, you don't need this at all and once the proxy is enabled, it will complain about missing pygeofilter dependency.

Reply:
that's very weird 🤔

Reply:
Can you post here lambda errors you got before adding this line?

Reply:
@amarouane-ABDELHAK 
This was the error log: 

```python
[ERROR] Runtime.ImportModuleError: Unable to import module 'handler': cannot import name 'parse' from 'pygeofilter.parsers.cql2_json' (unknown location)
Traceback (most recent call last):
```

Reply:
Tried by removing the dependency from `Dockerfile` and surprisingly it worked. 😅